### ISSUE TITLE
Benchmarks

### ISSUE THREAD

**Issue by imirkin** [NoUJoe/JSBD#1]
It would help people to evaluate this library if you had some benchmarks available.

I'd recommend including comparisons to

https://github.com/MikeMcl/big.js
https://github.com/MikeMcl/decimal.js
https://github.com/Yaffle/BigDecimal

The libraries aren't completely identical, but getting some baselines on common operations -- parse, stringify, round, add, multiply, divide would be interesting.

**Comment by NoUJoe** [comment]
Yeah absolutely! I shall get on that! I'll reply in this issue when I've done it.
